java/ql/src/experimental/Security/CWE/CWE-552/UnsafeUrlForward.qhelp

```diff
@@ -5,27 +5,45 @@
 
 
 <overview>
-<p>Constructing a server-side redirect path with user input could allow an attacker to download application binaries 
+<p>Constructing a server-side redirect path with user input could allow an attacker to download application binaries
 (including application classes or jar files) or view arbitrary files within protected directories.</p>
 
 </overview>
 <recommendation>
 
-<p>In order to prevent untrusted URL forwarding, it is recommended to avoid concatenating user input directly into the forwarding URL.</p>
+<p>Unsanitized user provided data must not be used to construct the path for URL forwarding. In order to prevent
+untrusted URL forwarding, it is recommended to avoid concatenating user input directly into the forwarding URL.
```

Comment:
Mention what should be done. For example,

"Instead, user input should be checked for path traversal using `..` sequences, taking care to remove any URL encoding first." ?

Reply:
I've added what should be done. Please validate. Thanks.